pom.xml

```diff
@@ -1471,6 +1474,100 @@
         </plugins>
       </build>
     </profile>
+    <!--
+      A profile that is identical to "might-produce-incrementals" but with "flatten-maven-plugin"
+      configured to interpolate variables in the "scm", "url", and "version" elements and keep all
+      other elements untouched. This is only intended to be used with repositories like "pom" and
+      "plugin-pom". If the POM from these repositories were flattened with the "oss" flatten mode,
```

Comment:
I wonder if upstream https://www.mojohaus.org/flatten-maven-plugin/flatten-mojo.html#flattenMode could use a new value, or is our usage just sufficiently weird to not be of interest to anyone else?

Reply:
I doubt anyone else would want to use this particular configuration.